### DISCUSSION TITLE
SYNTH SAMPLE WITH FX OF HACKTRIBE

### DISCUSSION THREAD

**Discussion by zolakfr** [bangcorrupt/hacktribe#329] [Ideas]
hello, is there any way to have the fx of hacktribe and sample of the synth e2 version?

**Comment by bangcorrupt** [comment]
Not really, yet.  You can extract the sample data from the synth as one long file, but I think there may be some bugs with the script.  Hacktribe won't correctly load the multisampled instruments, and doesn't have the overdrive effect on the Osc Edit knob.  I plan to add those in the future, but I don't know when.  If I can't make a fully hybrid firmware I will look at adding some of the hacks to the synth firmware.

[Dump-Synth-PCM-Samples](https://github.com/bangcorrupt/hacktribe/wiki/Dump-Synth-PCM-Samples)
#77 
#319

**Reply by apiel** [reply]
Having sampling capability on the e2 synth version but keeping all the rest from the synth version, would be really great. I would love to keep my e2 as it is and having sampling capability on top of it, would open so much possibilities...

**Reply by Rasha889** [reply]
That's was really incredible. Can I something do to Motivation you ? I can start with some simple tasks 😅
